powsybl-network-store-client-parent/pom.xml

```diff
@@ -173,17 +152,4 @@
         </pluginManagement>
     </build>
 
-    <profiles>
```

Comment:
What are those previous profiles used for ?

Reply:
for building the client